infrastructure/templates/search/parameters/main-dev.bicepparam

```diff
@@ -4,5 +4,5 @@ using '../main.bicep'
 param environmentName = 'Development'
 
 param contentApiUrl = 'https://content.dev.explore-education-statistics.service.gov.uk'
-param publicSiteUrl = 'https://dev.explore-education-statistics.service.gov.uk'
+param searchServiceIndexerName = '' // Overridden by the Azure Pipeline
```

Comment:
Aha - where I've had similar issues in the past I've always given the parameters in main.bicep a default value of '' rather than doing it in the env-specific param files.  Don't know if we have a preference either way, but doing it in main.bicep, you only have to do it once there!